### PR TITLE
docs: Risk monitoring example

### PR DESCRIPTION
**Pull request by allglc**

### CODE REVIEW

File: examples/exchangeability_testing/1-quickstart/plot_exchangeability_fixed_dataset.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+Currently, only `RiskMonitoring` is shown.
+"""
+
+from sklearn.linear_model import LogisticRegression
+
+from mapie.exchangeability_testing import RiskMonitoring
+from utils import generate_gaussian_stream, sample_two_gaussians
+
+##############################################################################
+# We first fit a classifier on clean training data. Then, in the same workflow,
```
Comment (GBrelurut) [inline examples/exchangeability_testing/1-quickstart/plot_exchangeability_fixed_dataset.py]: 'reference' instead of 'clean' ?
then 'reference test set' instead of 'clean reference set' ?

File: examples/exchangeability_testing/1-quickstart/plot_exchangeability_fixed_dataset.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+prop_shift = 0.5
+method_name = "RiskMonitoring"
+
+X_train, y_train = sample_two_gaussians(random_state=random_state)
+X_reference, y_reference = sample_two_gaussians(random_state=random_state + 1)
+
+clf = LogisticRegression(random_state=random_state)
+clf.fit(X_train, y_train)
+
+X_fixed_no_shift, y_fixed_no_shift = generate_gaussian_stream(
```
Comment (GBrelurut) [inline examples/exchangeability_testing/1-quickstart/plot_exchangeability_fixed_dataset.py:36]: Add vizualisation ?

File: examples/exchangeability_testing/1-quickstart/plot_exchangeability_online.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+"""
+
+from sklearn.linear_model import LogisticRegression
+
+from mapie.exchangeability_testing import RiskMonitoring
+from utils import generate_gaussian_stream, sample_two_gaussians
+
+##############################################################################
+# We first fit a classifier on clean training data. Then, in the same workflow,
+# we estimate the monitoring threshold on a clean reference set and update the
```
Comment (GBrelurut) [inline examples/exchangeability_testing/1-quickstart/plot_exchangeability_online.py]: Idem https://github.com/scikit-learn-contrib/MAPIE/pull/895/changes#r3073637627

File: examples/exchangeability_testing/2-advanced-analysis/plot_risk_monitoring.py [hunk 4]
```diff
@@ -0,0 +1,237 @@
+"""
+# Detect harmful shifts with RiskMonitoring
+
+In this example, we show how to use `RiskMonitoring` to track the
+misclassification risk of a deployed binary classifier on an online stream.
+The monitoring threshold is first estimated on a clean reference set, then
```
Comment (GBrelurut) [inline examples/exchangeability_testing/2-advanced-analysis/plot_risk_monitoring.py]: idem https://github.com/scikit-learn-contrib/MAPIE/pull/895/changes#r3073637627

**Review by GBrelurut — COMMENTED** [review]
TODO before merge : add modification of risk-monitoring implementation branch

**Review by GBrelurut — APPROVED** [review]

File: examples/exchangeability_testing/1-quickstart/plot_exchangeability_fixed_dataset.py (hunk 2 above)
Comment (allglc) [reply examples/exchangeability_testing/1-quickstart/plot_exchangeability_fixed_dataset.py:36]: yes, quickstart are currently just incomplete drafts as they need all methods implemented to work
